src/Templates/src/templates/maui-blank/.template.config/ide.host.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "icon": "ide/icon.ico",
+    "unsupportedHosts": [
+        {
+            "id": "vs",
+            "version": "(,17.1)"
```

Comment:
```suggestion
            "version": "(,17.2)"
```

Reply:
Only show the template for VS 17.2 and up

Reply:
This template was based on the `release/6.0.2xx-preview14` branch, so let me check this.